src/ontology/uberon.Makefile

```diff
@@ -1788,13 +1788,6 @@ $(COMPONENTSDIR)/in-subset.owl: $(SRC) $(TEMPLATEDIR)/in-subset.template.tsv
 $(COMPONENTSDIR)/vasculature_class.owl: $(TEMPLATEDIR)/vasculature_class.owl
 	$(ROBOT) merge -i $< annotate --ontology-iri $(ONTBASE)/$@ --output $@
 
-HRA_SUBSET_URL="https://raw.githubusercontent.com/hubmapconsortium/ccf-validation-tools/master/owl/UB_ASCTB_subset.owl"
-$(TMPDIR)/hra_subset.owl:
-	wget $(HRA_SUBSET_URL) -O $@
-
-$(COMPONENTSDIR)/hra_subset.owl: $(TMPDIR)/hra_subset.owl
-	$(ROBOT) merge -i $< annotate --ontology-iri $(ONTBASE)/$@ --output $@
```

Comment:
Probably you will have to keep these two rows, a bit changed, until ODK is fixed for components with `mirror_from`

Reply:
I won't merge this PR before the new ODK release.

Reply:
Are you sure - this could be.. a long time waiting!

Reply:
That's fine. It was just an update, and it doesn't change anything. This was a try because I'll need to add another component, so I'll do it the same way we have now.